lib/logqueue-fifo.c

```diff
@@ -353,12 +363,19 @@ log_queue_fifo_push_tail(LogQueue *s, LogMessage *msg, const LogPathOptions *pat
       log_msg_write_protect(msg);
       node = log_msg_alloc_queue_node(msg, path_options);
       iv_list_add_tail(&node->list, &self->input_queues[thread_index].items);
-      self->input_queues[thread_index].len++;
+      gsize qlen = self->input_queues[thread_index].len++;
```

Comment:
This could result in the following:
`qlen == 256` but due to the post-increment `++` the value is `257`. Is the extra item intentional? If not, how about the following:
```suggestion
      gsize qlen = ++self->input_queues[thread_index].len;
```

Reply:
I squashed the change to the initial commit